bin/octobox_bitbar.rb

```diff
@@ -104,11 +104,18 @@ def menu_string
+  IMAGE_INVERTED = "iVBORw0KGgoAAAANSUhEUgAAAEAAAABACAQAAAAAYLlVAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAAAmJLR0QA/4ePzL8AAAAJcEhZcwAAIxAAACMQAWcv3dYAAAAHdElNRQfiCRoHFRZjwoBjAAAGiElEQVRo3u2YW4wUVRrHf6eqenoWGQUyODiCgwq4sFxEuWhA1DWro2R1NJE1+KCiPniNSrxEvEWJMdFEjUYNiqBGHINxE41i1JjMrpsdvERFF3cXryHKgFwdZbqnuurvQ52uruquHiHR8NJ1Hrr6XL7v/92/OkYc2Mc5wPwbABoAGgAaABoAGgAOPADvtydpXFxA8o2LA4hAdau+qd8PGEM0QBgARDgEKReS68Yk3j1ClHW2LgCTQyplMHHkZ+z2MNG8yTGL+RzPNm5hHHPw2UqPwnoUM01g3IicaWYq05nEKHy2sIFebSMwOUppWYynEphW5rCe5ZzJNt7kGe01eyjSQhuTTZ92yC/vTD2qGeSE4Hge5YuqpR2sYqoQeIn9RnA0ayjyNMM4lJzgaO6hl38zRQhuJ2Qr3UwRuJgUtyrmBk8wlhfjqUGKFClQpGRn7heCpgpcLqOfXhbGVA6jh/+ylktoJU+eJk7jQXr4GweVIWcCwOAKzmcvQhQpEiaWQ3yKCPExHRGESBPMYKKlcDLvs6ZWqzGHW1khcDMBWOmvR4T8bFmX8FMgREAB0c8Jgj8IwV/LGuEehLjI0juKxdxHN6+yhns5lzYh2M3Siv7SAHKCJYmJEoP2za8SpIAQZwom8A0vCIYJLkf8nz8LQSev0F91qo8nGUEH6zmkooU0+1kI8U/u5jwmMJrJnMQK6wlpYpEpTuUxHrfnm5jOQjzBcbwTa6toxyCBnbkKQ77GBDhCsIPPmMcYIRjPi7zPa7RzHH0xy8qIIHUKQTOPEtAlBLfEpiuljBfEGn22wrECICd4iOcFT1DgG9q52C5FZHvrQCgyW/Ap4mfaBS9l6isJYwCxugoAruCP3CS4wh5fJpjGHJoFp3MF7fRmEB5EbGExm3iLdsE/EIU4XOuNAuKSON/EqWQEOUEnQnzPFMEI7uApZtLMKv7FjWzPcMcioo8OIehBDFTFTJgBoIT4jmGRFioArmUj3eSZyRKOEBjejcAIwTQWcA4irCFZRHxNO8/H0SFCAvw4gH3rgJXhI5bY0I2SCQvt0tKEf36FEBtw4qz3cIYnROT2JKT1E3oqaySoOfGywMGUHfAmq5rFghbu4j2u5VheZjVHChw8a7HPMl3MRxTtfARwF91czEw6mMhSClUQfMT/aBK4NiAYy7v8YLP8Y3bbMckQtUDnxlas9u5I8gCxi5ujrBfr8mqUyqglxE7GCLw4CgQtQtDGAD4FxDRBc5l9vOs6xGCmc0WSPRelGXLkcHHICybip2CXED8xXuA5YBwF5lyzSP0mD+xiMx55HtGnxlVBoa35jgJzknmF+WwlR5jZxThAMwfbOk/it7bvCQnsIcFw9iJmWEmnsowLytGRUH+n/XMn22vcKimbz3IOS5ng0ioT+Ig+Rgm8smVLiHsFw3mbTxhZVTKNwOFzhPhScFVmNCRT9B5W00UHLYxkMXuq/MZH/AdX4EYATiRA/F1wPkK8XdXz5AUOHyA+jFyTtYiBOhBKMbi9fMfOOmH4qg1DXMFR9CPesOV4gM0MTxTM8u8w5saQRvL1EFqISnmZaZCZiO6sJCJji83TgkmUEBtxcGjCsXViNi+xgU575BS+4lmmEFKbmtMjJMiMlxAxvwLAFZzGl8wRgrNZx+yU8stZ8kYhOAMhVgoWIcJfLT3Zjrox9q0qbzdCMJ6VrOYahgtaraLXYQRt/Ih4Bwdja+f+Qygirq+qhjiClSyzAXeR3bqJcbTwAV+wPKoHTGcXT9Es8PBsWhqq/mdHybc0x8JaAJ6gG/GI1cQN/ECIuFLgxcWonRab59y4h44al0Kd3Firfh/RVZY/3RFNYBBxm60ObfyFE8rbBGN5g92MFeTiuDDkBPP4PhV8Q7EvIB4QeGWzx2t4gssQYkHKKzzygi6EOD3ZUNv1JsEhrLHqHSoqIoArRKXApdtyR3Aen7NIcCjH8CfGWTvNQ6xjTDo/xuciZzqLj22I+fiUCOwoUUo0Jzen2Vd/GZUL70Q22andrGMcrUyusMqA4NlKeSHrUwtpv3idGQInyV7pz3NjcKPLBHMEs5jEaPJsY5U2g3EwCoa4S3Cj714zly4WMInWuOrtYBM9rNVHYLzqy4ohLihSn+tSuA/74g93M5rDGYVHkZ1s0XZLhVoR6gAoX7MADv6+ME+AMBqsmW1CWRcb+6iB/X2ME9U5sEKE9UX4XQDsz3PAr+kaABoAGgAaABoAfgGUfyPTDMZ7qQAAACV0RVh0ZGF0ZTpjcmVhdGUAMjAxOC0wOS0yNlQxMToyMToyMi0wNDowMOFilSMAAAAldEVYdGRhdGU6bW9kaWZ5ADIwMTgtMDktMjZUMTE6MjE6MjItMDQ6MDCQPy2fAAAAAElFTkSuQmCC"
   TMP_DATA_FILE = '/tmp/octobox-bitbar-ids.json'.freeze
   MARK_READ_AND_OPEN = File.join(__dir__, 'octobox_mark_read_and_open.rb')
   ARCHIVE = File.join(__dir__, 'octobox_archive.rb')
 
+  def terminal_image
+    # Default is mostly white, so use inverted for "Light" theme
```

Comment:
I somewhat disagree - The octobox favicon is black, and though `templateImage` seems to work with your white image, the [docs](https://github.com/matryer/bitbar#plugin-api) ask for black-only.  Can you switch which you call inverted?